<?xml version="1.0"?>
<launch>

	<node pkg="ipa_dirt_detection_dataset_tools" type="simple_segmentation" name="simple_segmentation" output="screen">
		<rosparam command="load" file="$(find ipa_dirt_detection_dataset_tools)/launch/simple_segmentation_params.yaml"/>
	</node>

</launch>
